### PR TITLE
Feature homepage

### PR DESCRIPTION
**Pull request by andela-tadesanya**
Please merge my branch so I can delete the feature-homepage branch.

Thanks.

### CODE REVIEW

File: troupon/deals/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        driver.get("http://127.0.0.1:8000/")
+        assert "About Troupon" in driver.page_source
+
+    def test_social_links(self,):
+        """
+        Checks social links are working
+        """
+        driver = self.driver
+        driver.get("http://127.0.0.1:8000/")
+        assert "http://facebook.com/troupon/" in driver.page_source
```
Comment (andela-cnnadi) [inline troupon/deals/tests/test_views.py]: This test is failing

**Comment by andela-cnnadi** [comment]
Check this tutorial to see how we can do it in such a way that we don't have to always startup the server manually before running the test -> http://agiliq.com/blog/2014/09/selenium-testing/ using LiveServerTestCase.

File: troupon/deals/tests/test_views.py (hunk 1 above)
Comment (andela-tadesanya) [inline troupon/deals/tests/test_views.py]: Yes it will, until both deals and social links are added to the homepage HTML.

Comment (andela-cnnadi) [inline troupon/deals/tests/test_views.py]: So remove the test until you add the functionality.

**Comment by andela-tadesanya** [comment]
I have implemented LiveServerTestCase in the tests and removed tests that are going to fail.

**Pull request merged**